setup.py

```diff
@@ -168,15 +168,12 @@ def build_dir():
     "MurmurHash3"])
 
 # Don't forget to update lib/Makefile with these flags!
-EXTRA_COMPILE_ARGS = ['-O3', '-std=c++11', '-pedantic',
-                      '-fno-omit-frame-pointer']
-EXTRA_LINK_ARGS = ['-fno-omit-frame-pointer']
+EXTRA_COMPILE_ARGS = ['-O3', '-std=c++11']
+EXTRA_LINK_ARGS = []
```

Comment:
Some collateral damage from my debugging, but after reading up a bit it's probably better we didn't enable those flags unless we have a compelling reason to, in which case, documenting it would be great!